src/main/java/core/basesyntax/figure/Circle.java

```diff
@@ -0,0 +1,53 @@
+package core.basesyntax.figure;
+
+import core.basesyntax.tools.Color;
+
+public class Circle extends Figure {
+    private int radius;
+    private double area;

```

Comment:
```suggestion
```
let's remove this field

the problem is, when we store the value of area - if we change radius, we will have to recalculate area as well
this takes additional effort that we can avoid

Reply:
not fixed